addons/mail/models/mail_thread.py

```diff
@@ -275,7 +275,7 @@ def create(self, vals_list):
             for key, val in self._context.items():
                 if key.startswith('default_') and key[8:] not in create_values:
                     create_values[key[8:]] = val
-            thread._message_auto_subscribe(create_values)
+            thread._message_auto_subscribe(create_values, existing_policy='update')
```

Comment:
maybe prefix naming with `followers` to clarify its purpose?

Reply:
@alexkuhn Changes are done.